hack/externalnode/install-vm.ps1

```diff
@@ -167,41 +174,102 @@ function UpdateAgentConf() {
             Log "Updating $AntreaAgentConfPath with ${ExternalNodeNamespace}: ${Namespace}"
             [System.IO.File]::AppendAllText($AntreaAgentConfPath, "  ${ExternalNodeNamespace}: ${Namespace}" +
                     ([Environment]::NewLine))
+        } elseif ($line -like "*$Bridge*") {
+            Log "Updating $AntreaAgentConfPath with ${Bridge}: ${OVSBridge}"
+            [System.IO.File]::AppendAllText($AntreaAgentConfPath, "${Bridge}: ${OVSBridge}" +
+                    ([Environment]::NewLine))
         } else {
             [System.IO.File]::AppendAllText($AntreaAgentConfPath, $line +
                     ([Environment]::NewLine))
         }
     }
 }
 
-function ConfigureAntreaAgentService() {
-    # Set environment variables
-    [Environment]::SetEnvironmentVariable("NODE_NAME", $NodeName, [System.EnvironmentVariableTarget]::Machine)
-    # Assume nssm is installed and configure service
-    $AntreaAgentArgs = "--config $AntreaAgentConfPath --log_file $AntreaAgentLogFile --logtostderr=false"
-    log "Creating service $AntreaAgent $AntreaAgentPath $AntreaAgentArgs"
+function CreateAntreaAgentStartupScript() {
+    $Script:StartAntreaAgentScript = "$AntreaAgentConfDir\Start-AntreaAgent.ps1"
+    $StartAntreaAgentScriptContent = '
+Param(
+    [parameter(Mandatory = $true)] [string] $OVSBridge,
+    [parameter(Mandatory = $true)] [string] $InstallDir
+)
+
+$AntreaSwitch = "antrea-switch"
+$AntreaAgentConfDir = [io.path]::combine($InstallDir, "conf")
+$AntreaAgentLogDir = [io.path]::combine($InstallDir, "logs")
+$AntreaAgentConfPath = [io.path]::combine($AntreaAgentConfDir, "antrea-agent.conf")
+$AntreaAgentLogFile = [io.path]::combine($AntreaAgentLogDir, "antrea-agent.log")
+$AntreaAgentPath = [io.path]::combine($InstallDir, "antrea-agent.exe")
+$InstallLog = [io.path]::combine($AntreaAgentLogDir, "install_vm.log")
```

Comment:
Is it for antrea-agent service log?

Reply:
The install_vm log will capture the logs from the Powershell scripts.
- Includes logs from install-vm.ps1 and Start-AntreaAgent.ps1

This will help to troubleshoot.

Reply:
I agree that the logs are useful to troubleshoot, but it is strange to output the debuging log to a file named as "install_vm.log" when user restart antrea-agent service or after system startup since the service operation is not about installation at that time.

Reply:
Discussed offline, we decided to change the log file name to antrea-agent-service.log.